packaging/hudi-hadoop-mr-bundle/pom.xml

```diff
@@ -48,7 +48,7 @@
               <goal>shade</goal>
             </goals>
             <configuration>
-              <createSourcesJar>true</createSourcesJar>
+              <createSourcesJar>false</createSourcesJar>
```

Comment:
Does this mean sources jar will not be getting generated. Instead, can you enable sources jar for non-bundle packages like hudi-common, hudi-client, .....  ?